examps/Adder/AdderStAut.txs

```diff
@@ -54,7 +54,11 @@ MODELDEF  Adder ::=
     BEHAVIOUR  adder [ Action, Result ] ( )
 ENDDEF
 
-
+MODELDEF  STD_Adder ::=  -- TODO remove in final version
+    CHAN IN    Action
```

Comment:
Minor: I prefer to have issues instead of TODO's.

Reply:
Normally I agree.
Yet these todos should be removed soon.... when only the new stautdef remains.